### ISSUE TITLE
Install errors with Node v12+ on Win10

### ISSUE THREAD

**Issue by faniry6** [artutra/OpenChord#105]
By ```npm install``` I got the following error on the latest update

```
......
 error C2039: 'NamedPropertyG
etterCallback': is not a member of 'v8'
......
```

By a simple google search, it seems like downgrading Node to v10 would solve this issue. Did you get the same issue? which Node version is needed to build the project?

Thanks again.

**Comment by faniry6** [comment]
Using Node v10.18.0 seems to work but have 1 high vulnerabilities

**Comment by artutra** [comment]
I am using Node v10.15.1 to build this project. Could yout try again from the master branch? I've just merged some changes #106

**Comment by faniry6** [comment]
I updated the branch, I still got the following incompatibility

```
eslint-plugin-react@7.12.4 requires a peer of eslint@^3.0.0 || ^4.0.0 || ^5.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN eslint-plugin-react-native@3.6.0 requires a peer of eslint@^3.17.0 || ^4 || ^5 but none is installed. You must install peer dependencies yourself.
npm WARN react-native-fs@2.16.6 requires a peer of react-native@^0.59.5 but none is installed. You must install peer dependencies yourself.
npm WARN react-native-fs@2.16.6 requires a peer of react-native-windows@^0.57.2 but none is installed. You must install peer 
dependencies yourself.
```
Normally I have to solve all these manually.

**Comment by artutra** [comment]
Closed on v0.5.0
